.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,45 @@
+name: Integration Tests
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  selenium-tests:
+    name: Selenium Tests
+    runs-on: ubuntu-latest
+    env:
+      PROJECT: wordpress-plugin
+      ORGANIC_DEMO_SITE_APIKEY: ${{ secrets.ORGANIC_DEMO_SITE_API_KEY }}
+      COMPOSER_AUTH: '{"github-oauth":{"github.com":"${{ secrets.GITHUB_TOKEN }}"}}'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up python
+        id: python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Install poetry
+        run: |
+          curl -sSL https://install.python-poetry.org | python3 -
+
+      - name: Configure poetry
+        run: |
+          poetry self add poetry-dotenv-plugin
+          poetry install
+
+      - name: Build the stack
+        run: |
+          mv .env.ci .env
+          poetry run ./dev.py up
```

Comment:
Do we want to run the tests for all WP services? I've only done the default WP service here.

Reply:
Yes, best case would be to run them in parallel

Reply:
I will also look into [caching dependencies](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows) as well.

Reply:
I'm going to add support for wp49-php72 later:

- Tests don't currently work for WP < 5 (pre-Gutenberg). (Well, actually they fail. But it seems pointless to update the tests at the moment if there are no other tests to run.)
- There was an issue with installing the WP theme. (It's somewhere in the failed Actions--I was going to link to it, but it's getting late.)